### ISSUE TITLE
A better payment method

### ISSUE THREAD

**Issue by Omarlatif** [CraftingStore/Suggestions#644]
**Is your feature request related to a problem? Please describe.**
It is a tad bit frustrating when you have to add another month to the CraftingStore plan.

**Describe the solution you'd like**
Two solutions can be done regarding this, either; Set up automatic payments to automatically increase the days when getting low, or an option to add more than just 1 month of benefits... Example: 1 month, 3 months, 6 months, 1 year.

**Additional context**
None.

**Comment by Sikatsu** [comment]
Agreed, this system can definitely use an improvement. We really just need an option to pay for like half a year or even a year.

**Comment by Tim-NL** [comment]
Hi,

For the latter suggestion; You can top it up a few times in a row. It will add one month every time. So if you'd upgrade 3 times, you'd get 3 months of service. I'll leave the suggestion open as we may be able to do a more "user-friendly" option.

-Tim

**Comment by Omarlatif** [comment]
> 

That is indeed possible, but it does get quite frustrating when you add multiple months one by one. It would be much easier and more user-friendly to add an option to add multiple months.

**Comment by Sikatsu** [comment]
I know that is possible, though that is not the best option and as mentioned above it can get rather frustrating..

**Comment by creulcat** [comment]
Having an option to just do it in one go would be really nice.
It also helps with the bank statements / balancing finances eventually, just knowing your yearly/quarterly/etc expenses.
